test/integration/test_integration.py

```diff
@@ -113,6 +113,9 @@ def results(request):
 _docutils = fixtures_factory('python-docutils-0.13.1-4.fc26')
 docutils = fixtures_factory('_docutils')
 
+_nodejs = fixtures_factory('nodejs-semver-5.1.1-2.fc26')
+nodejs = fixtures_factory('_nodejs')
```

Comment:
Could you please add the new fixture to the first test, so all the downloading etc. happens at the beginning and all other tests happens at once?